src/main/java/redis/clients/jedis/ShardedJedis.java

```diff
@@ -206,20 +207,16 @@ public String psetex(final String key, final long milliseconds, final String val
     return j.psetex(key, milliseconds, value);
   }
 
-  public List<String> blpop(final String arg) {
```

Comment:
@gkorland @dengliming I've now removed this method, because it's unusable.

Reply:
@sazzad16 isn't it a backward that will break applications runtime? why not deprecate it?

Reply:
@gkorland Programmatically, yes, it's breaking. But functionally, it's broken since the moment it was created. It's unusable. Trying to use it will always generate error from Redis. Because this command requires at least two arguments and there's no way to pass those. I don't believe there's a sane application that has this in their code. That's why I removed it. (Even though I deprecated it first.)